amplipi/models.py

```diff
@@ -739,6 +753,12 @@ class Config:
       },
     }
 
+
+class ImageSpec(BaseModel):
+  """ Convert image found at @uri to a square image with @height """
+  height: int = Field(120, ge=40, le=300, description="Image height (and width)")
```

Comment:
Does this need to match the other set of ge/le?

Reply:
This was a relic of an intermediate design good catch, removing...